features.json

```diff
@@ -153,17 +153,19 @@
 		"Node.js": {
 			"url": "https://nodejs.org/",
 			"logo": "/images/nodejs.svg",
-			"version": "15.0",
+			"version": "16.4",
 			"features": {
 				"bigInt": true,
 				"bulkMemory": true,
+				"exceptions": "--experimental-wasm-eh",
 				"multiValue": true,
 				"mutableGlobals": true,
-				"referenceTypes": "--experimental-wasm-anyref",
+				"referenceTypes": "--experimental-wasm-reftypes",
```

Comment:
it was renamed year ago https://github.com/v8/v8/commit/ba688c6ec9c0fb4439f7381cb8046ea14b159d0e#diff-73af4fa91a5bf27a89d47cce1271322131f71f54223cea49939723a72f5d3f6e